### DISCUSSION TITLE
Taskbar toolbar does not show icons

### DISCUSSION THREAD

**Discussion by DBadke** [valinet/ExplorerPatcher#3790] [Q&A] [answered]
Windows 11 build 22631.4317, ExplorerPatcher version 22621.3880.66.6
Taskbar is at the left edge of monitor, auto hide, stretched wider, taskbar locked

Problem: I created a taskbar toolbar, pointed it to a folder containing shortcuts (imaginatively called "Toolbar"), the toolbar appears where it should, but all that is visible is a "Toolbar" label and a >> icon.

![toolbar1](https://github.com/user-attachments/assets/dd215ede-f6e1-48fb-b924-8ea94545ed26)

 When the >> is clicked a drop down list appears with the shortcuts in the folder.

![toolbar2](https://github.com/user-attachments/assets/e6f2653f-3497-4ce7-b020-e6f10ad3b531)

What I want is a row of icons, like the Windows 10 taskbar toolbar.

![toolbar3](https://github.com/user-attachments/assets/d38f9c5d-9a11-4923-810f-620cb760001e)

I have been unable to find any way to fix this. The drop down list is better than nothing, but the row of icons would be much better. Is there any way to do this?

David

**Comment by pyrates999** [comment]
If you haven't yet, open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

**Reply by DBadke** [reply]
It is set to Windows 10 (ExplorerPatcher). Still no icons on the toolbar...

**Reply by pyrates999** [reply]
What if you move the taskbar to the bottom of the screen?

**Reply by DBadke** [reply]
Bottom, top, left, right, on monitor 1 and monitor 2... still no icons. If it helps, here is the settings export file.

[ExplorerPatcher_22621.3880.66.6.txt](https://github.com/user-attachments/files/17350850/ExplorerPatcher_22621.3880.66.6.txt)

**Reply by pyrates999** [reply]
Please wait for a developer to respond.

**Comment by Amrsatrio** [comment, marked as answer]
Unlock your taskbar. Stretch the handle for the toolbar you've just created.

**Reply by DBadke** [reply]
Well, that was easy! Stretched the toolbar area, and the icons magically appeared. Thanks!!!

Still not perfect, though. The toolbar shows large icons and a label for each button.

![Toolbar](https://github.com/user-attachments/assets/314eb0d8-19e8-408a-9ed4-d3535ce69658)

I would like it to look like the tray icons. Setting the taskbar to "Combine taskbar buttons and no labels" doesn't affect the toolbar, and since Windows 11 doesn't want toolbars at all, there seems to be no way in settings to change it. Is there any way to shrink the icons and lose the labels? Even a registry hack would be OK.

**Reply by pyrates999** [reply]
I think if you right click on the toolbar, there is an option to not show text.

**Reply by DBadke** [reply]
Right clicking on the toolbar label or any empty area of the toolbar just shows the standard taskbar menu, which doesn't have any toolbar options. Right clicking on an icon shows the explorer menu, which is bloated with useless stuff but nothing about the toolbar. Since Microsoft no longer believes in toolbars, I don't think there is any Windows setting that will work, but perhaps some hack would, through ExplorerPatcher or something else..

**Reply by pyrates999** [reply]
Right click on an empty area of the toolbar you created on the taskbar.

**Reply by DBadke** [reply]
I did right click on an empty area of the taskbar toolbar. That just displays the taskbar menu. Do you get a different menu, or is there something I am missing on the taskbar menu?

![Toolbar](https://github.com/user-attachments/assets/a44551b0-8f3c-4c04-8116-aba828d6d9e0)

**Reply by pyrates999** [reply]
According to this: https://www.tenforums.com/tutorials/4624-add-remove-quick-launch-toolbar-windows-10-a.html

There should be **Show Text** and **Show Title**.

**Reply by DBadke** [reply]
I just found the menu items, and they work! You have to right click directly on the _**separator bar**_ above (or to the left/right of) the toolbar when the taskbar is unlocked. Clicking on an empty area or anywhere else does not work. Now I have a proper toolbar again.

![toolbar4](https://github.com/user-attachments/assets/99984dee-23c5-40e3-93a0-beedef131709)

Thanks for all the help! I might never have figured this out on my own.

David
